test/system/260-sdnotify.bats

```diff
@@ -371,4 +371,100 @@ READY=1" "sdnotify sent MAINPID and READY"
     run_podman rmi $(pause_image)
 }
 
+# The functione takes the following arguments:
+# 1. File to write to
+# 2. Command container 1
+# 3. Command container 2
+# 4. Sdnotify policy
+function generate_exit_code_yaml {
+    echo "
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+  annotations:
+    io.containers.sdnotify: "$4"
+spec:
+  restartPolicy: Never
+  containers:
+    - name: ctr1
+      image: $IMAGE
+      command:
+      - $2
+    - name: ctr2
+      image: $IMAGE
+      command:
+      - $3
+" > $1
+}
+
+@test "podman kube play - exit-code policy" {
+    fname="$PODMAN_TMPDIR/test.yaml"
```

Comment:
It's better to use a random file name to avoid possible issues when running locally